frontend/src/app/features/map/siteDrawer/SiteDetailsDrawerContent.tsx

```diff
@@ -0,0 +1,157 @@
+import { FC, RefObject, useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { Map } from 'leaflet';
+import {
+  useMapSearch_FindSiteBySiteIdQuery,
+  MapSearch_FindSiteBySiteIdQuery,
+} from '../../../../graphql/generated';
+import {
+  MagnifyingGlassPlusIcon,
+  ShoppingCartIcon,
+  SpinnerIcon,
+} from '../../../components/common/icon';
+import './MapSearchDrawer.css';
+import { Button } from '../../../components/button/Button';
+import AddToFolio from '../../folios/AddToFolio';
+import { getZoom, MAP_FLY_OPTIONS } from '../mapOptions';
+import { MapSearchQueryParamsContext } from '../mapSearchQueryParamsContext/MapSearchQueryParamsContext';
+
+const SummaryItem = ({
+  label,
+  value,
+}: {
+  label: string;
+  value?: number | string | null;
+}) => {
+  return (
+    <div className="summary-item">
+      <div className="fw-bold">{label}</div>
+      <div>{value || ''}</div>
+    </div>
+  );
+};
+
+type Site = MapSearch_FindSiteBySiteIdQuery['findSiteBySiteId']['data'];
+const formatAddress = (site: Site) => {
+  const addressLines = [
+    site?.addrLine_1,
+    site?.addrLine_2,
+    site?.addrLine_3,
+    site?.addrLine_4,
+  ];
+  return addressLines.filter(Boolean).join(', ');
+};
+
+const formatCoordinates = (coords: Array<number | undefined | null>) => {
+  const [d, m, s] = coords;
+  let result = '';
+  if (d) result += `${d}d`;
+  if (m) result += `, ${m}m`;
+  if (s) result += `, ${s}s`;
+  return result;
+};
+
+interface SiteDetailsDrawerContentProps {
+  mapRef: RefObject<Map | null>;
+}
+export const SiteDetailsDrawerContent: FC<SiteDetailsDrawerContentProps> = ({
```

Comment:
This component is mostly unchanged compared to its previous iteration - `SiteDetailsDrawer`. The _only_ difference is that it doesn't render its own `<Drawer>` component anymore.

Everything else is left unchanged.